src/styles/layouts/_container.scss

```
@@ -1,12 +1,13 @@
 @use "../abstracts" as *;
 
 .l-container {
-	max-width: 121.6rem;
+	max-width: calc(121.6rem + $spacer-sm * 2);
```

```suggestion
	max-width: 121.6rem + ( $spacer-sm * 2 );
```

calc is hier onnodig, `rem + rem * factor` is statisch compileerbaar.